src/sentry/static/sentry/app/views/performance/vitalDetail/table.tsx

```diff
@@ -210,7 +210,10 @@ class Table extends React.Component<Props, State> {
       );
     }
 
-    if (field.startsWith('key_transaction') || field.startsWith('user_misery')) {
+    if (
+      field.startsWith('key_transaction') ||
+      field.startsWith('user_misery_prototype')
```

Comment:
Does this check for `user_misery*` do anything? I don't think `user_misery` was ever in the vital details page, so this might be dead code.

Reply:
I think you are right, I didn't notice any visible differences so...